tools/pipelines/templates/build-npm-package.yml

```diff
@@ -585,6 +589,10 @@ stages:
                 exit -1;
               fi
 
+      # Job - Additional build steps
+      - job: addtional build
+        - ${{ parameters.additionalBuildSteps }}
+
```

Comment:
The additional build steps need to be in the same job as the tasks above; if we put them in a new job, that will run in a new agent that doesn't even have the repository checked out (we'd need to add steps specifically for that, then all the steps to build it again, etc).

Looks like the indentation of line `- ${{ parameters.additionalBuildSteps }}` already matches the one for the tasks above, so we just need to remove lines 592 and 593.

Reply:
Oh! Like this?